### PR TITLE
Remove cclass decorator from cython snippet in userguide

### PR DESCRIPTION
**Pull request by touilleMan**

### CODE REVIEW

**Comment by scoder** [comment]
Thanks. The empty lines are there to keep the code lines of both the Cython-ish and Python-ish variants next to each other. We should keep that.

**Comment by touilleMan** [comment]
@scoder thanks for the quick feedbacke ! I've updated my PR ;-)

File: docs/src/userguide/extension_types.rst [hunk 1]
```diff
@@ -621,7 +621,6 @@ usually set on an extension type or C method using a decorator:
                def describe(self): pass
 
 
-            @cython.cclass
```
[the remark this answers is not in the document]
Comment (touilleMan) [reply docs/src/userguide/extension_types.rst:624]: I've updated the PR ;-)

[the remark this answers is not in the document]
Comment (0dminnimda) [reply docs/src/userguide/extension_types.rst:624]: Great! I think we can merge this

**Comment by da-woods** [comment]
I'm just going to let the CI run before merging. I'm pretty sure it's fine, but Python+Cython does depend on whitespace so it isn't 100% obvious the new code is right.

**Comment by da-woods** [comment]
Thanks

**Pull request merged**